### PR TITLE
Updated the deployment advisors page

### PR DESCRIPTION
**Pull request by v-lorm**
Made several small changes to the language and grammar.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/deployment-advisors-for-office-365.md [hunk 1]
```diff
@@ -17,31 +17,31 @@ search.appverid:
...
 
-As you select the features and options you want to deploy, the wizard builds a step-by-step setup plan that's customized to your needs. You'll get a full set of instructions, videos, reference articles, and scripts. Some wizards have automation that will change some of your settings, while in other areas of the wizards you change your settings and data with the wizard to guide you. You can use these wizards anytime, even during planning or after you've already set up the services, to learn more about Office 365 features and options.
+As you select the features and options you want to deploy, the advisor builds a step-by-step setup plan that's customized to your needs. You'll get a full set of instructions, videos, reference articles, and scripts. Some advisors have automation that will change some of your settings, while in other areas of the advisor you change your settings and data with the advisor to guide you. You can use these advisors anytime, even during planning or after you've already set up the services, to learn more about Office 365 features and options.
```
Comment (nenonix) [inline Enterprise/deployment-advisors-for-office-365.md]: while in other areas of the advisor you **can** change your settings and data with the advisor to guide you. You can use these advisors **any time**,

Comment (v-lorm) [reply Enterprise/deployment-advisors-for-office-365.md]: I don't think either change needs to be made. I'm fine with the change to "you can change." I disagree with the change to "any time". This is an informal discussion and anytime is indefinite; it implies at any time whatsoever. Go ahead and make both changes if the changed wording more closely aligns with the other docs articles.

Comment (JohanFreelancer9) [reply Enterprise/deployment-advisors-for-office-365.md]: Copy edit - agree with "can change", but would also prefer "anytime".

Comment (robmazz) [reply Enterprise/deployment-advisors-for-office-365.md]: I'm fine with these changes, "anytime" is also OK with me. Merging and closing.

**Pull request merged**